### PR TITLE
Bumped gsgenclient to latest version and adjusted usage after breaking change

### PR DESCRIPTION
**Pull request by axbarsan**
Towards https://github.com/giantswarm/giantswarm/issues/14617

### CODE REVIEW

File: commands/types/types.go [hunk 1]
```diff
@@ -79,7 +79,13 @@ type MasterDefinition struct {
 
 // MasterNodes defines an interface for configuring HA master nodes.
 type MasterNodes struct {
-	HighAvailability bool `yaml:"high_availability,omitempty"`
+	HighAvailability  bool             `yaml:"high_availability,omitempty"`
+	AvailabilityZones []string         `yaml:"availability_zones,omitempty"`
+	Azure             MasterNodesAzure `yaml:"azure,omitempty"`
```
Comment (axbarsan) [inline commands/types/types.go:84]: This should be a ptr

File: commands/types/types.go [hunk 2]
```diff
@@ -79,7 +79,13 @@ type MasterDefinition struct {
 
 // MasterNodes defines an interface for configuring HA master nodes.
 type MasterNodes struct {
-	HighAvailability bool `yaml:"high_availability,omitempty"`
+	HighAvailability  bool             `yaml:"high_availability,omitempty"`
```
Comment (axbarsan) [inline commands/types/types.go:82]: Should be a ptr, also

File: commands/types/types.go (hunk 1 above)
Comment (marians) [reply commands/types/types.go:84]: What is a ptr?

Comment (axbarsan) [reply commands/types/types.go:84]: This is a note for myself, the PR is not up for review

Comment (axbarsan) [reply commands/types/types.go:84]: ptr = pointer

**Pull request closed without merging**